cypress/integration/404Page.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('404 page testing', () => {
+  it('should return 404-page status', () => {
```

Comment:
Let's use `test` instead of `it`

Reply:
@nadezhdakharchuk can we use `specify` instead of it? Cypress [don't recognize](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-Structure) `word`